CHANGES.md

```diff
@@ -18,7 +18,7 @@ Format of the entries must be.
 
 ### Fixed and improved
 
-* Updated to [DC/OS UI 1.10.9-rc2](https://github.com/dcos/dcos-ui/releases/tag/1.10+v1.10.9-rc2)
+* Updated to [DC/OS UI v1.10.9-rc3](https://github.com/dcos/dcos-ui/releases/tag/v1.10+v1.10.9-rc3)
```

Comment:
Is it correct to have that in the 1.10.9 section?

Reply:
AFAIK 1.10.8 is the latest 1.10 release and we are working towards 1.10.9, thus we put that line under 1.10.9 changes. Am I missing something?

Reply:
Looks like :)

Reply:
Oh, I didn't see your response!

My response was a response to my question. So, don't worry!